app/views/arms/create.js.coffee

```diff
@@ -24,7 +24,7 @@ $('.form-error').remove()
 
 <% @errors.messages.each do |attr, messages| %>
 <% messages.each do |message| %>
-$("[name='arm[<%= attr.to_s %>]']").parents('.form-group').removeClass('is-valid').addClass('is-invalid').append('<small class="form-text form-error"><%= message.capitalize %></small>')
+$("[name='arm[<%= attr.to_s %>]']").parents('.form-group').removeClass('is-valid').addClass('is-invalid').append("<small class='form-text form-error'><%= message.capitalize.html_safe %></small>")
```

Comment:
These changes are refactors to improve handling of the error messages if they have HTML tags - you can skip all of the rest of these types of changes